capstone/timeline.md

```diff
@@ -5,43 +5,45 @@ subtitle: For MDS Capstone Projects
+| [Suggested initial submission of proposal](#proposal-revision-process)                         | November 1, 2022      |
+| [Proposal deadline](#proposal-revision-process)                                                | November 30, 2022     |
+| [Selection of proposals](#selection-of-proposals)                                              | December 15, 2022     |
+| [Partners submit video presentations](#partners-submit-video-presentations)                    | January 11, 2023      |
+| [Capstone fair](#capstone-fair)                                                                | January 20 2023       |
+| [Students rate the projects](#students-rate-the-projects)                                      | End-January, 2023     |
+| [Partners demonstrate readiness of data and sign legal docs](#partners-demonstrate-readiness-of-data-and-sign-legal-docs)                                    | February 1, 2023      |
+| Students are assigned to projects                                                              | Feb - March, 2023     |
+| [Kickoff meetings](#kickoff-meetings)                                                          | April, 2023           |
+| [Project](#project)                                                                            | May 1 - June 29, 2023 |
+| [Kickoff hackathon](#hackathon)                                                                | May 1 - May 3, 2023   |
 
 <br>
 
 ## Proposal revision process
 
 Members of an organization interested in participating as a capstone partner should submit a project [proposal](https://ubc-mds.github.io/capstone/proposal/).
 
-Proposals received *before November 15th* will be reviewed by the MDS teams and feedback/suggestions for revisions will be provided to the partner, who can then revise and re-submit their proposal. This revision process aims to maximize the chance of a proposal being selected by students. Proposals received *after November 15th* will not be provided with feedback.
+Proposals received *before November 1st* will be reviewed by the MDS teams and feedback/suggestions for revisions will be provided to the partner, who can then revise and re-submit their proposal. This revision process aims to maximize the chance of a proposal being selected by students. Proposals received *after November 1st* will not be provided with feedback.
 
 ## Selection of proposals
 
 The MDS leadership team will select proposals based primarily on student interest, and will invite the successful organizations to be a capstone partner.
 
-## Partners accept invitations
-
-The successful capstone partners accept the offers, and confirm their attendance at the capstone fair.
+## Partners submit video presentations
+Capstone partners should submit video presentations (5-10 minutes) to provide information additional to the proposals. This will help students review your project(s) before attending the capstone fair.
```

Comment:
@kvarada  @zmx721 

I wonder if we should mention something about the quality of the production here. The way we have it might be a little intimidating for companies that never produced a video before.  It might get discouraged if they think their video won't be a great production or if it would be too much trouble to make it a great quality video. 

I don't know. What do you think?

Reply:
I think it's a great idea to say something about the quality. How about something like this? 

> Capstone partners should submit video presentations (5-10 minutes) to provide information additional to the proposals. We are not expecting production quality videos and we do not want you to spend too much time on this. The main purpose here is helping our students review your project(s) before attending the capstone fair so that they are ready to ask you meaningful questions.

Reply:
Couldn't phrase it better! 👌

Reply:
Thank you @Lourenzutti and @kvarada. I have made the suggested change in [7b2503e](https://github.com/UBC-MDS/UBC-MDS.github.io/pull/136/commits/7b2503e281d488fab390463633b5318e0caadc3a) .